lib/std/enummaps.nim

```diff
@@ -0,0 +1,64 @@
+##[
+experimental module, unstable API
+]##
```

Comment:
I think we would want a license statement here, you can see other modules for a template.

Reply:
isn't that implied by top-level file `copying.txt` ? lots of files dont' have any license (and this could be done in a single cleanup PR to fix all files w missing license, if needed)

I'm not against your suggestion but this should be in https://nim-lang.github.io/Nim/contributing.html as an accepted guideline if that's indeed what should be done, can you file a PR in https://github.com/nim-lang/Nim/blob/devel/doc/contributing.rst (I don't mean to introduce red tape, but guidelines **save time** in the end by removing arguments)